doc/source/python/python_wrapping_s2i.md

```diff
@@ -249,9 +249,9 @@ class MyModel(object):
         return [{"type": "COUNTER", "key": "mycounter", "value": 1}]
 ```
 
-For more details on custom metrics and the format of the metric dict see [here](../custom_metrics.md).
+For more details on custom metrics and the format of the metric dict see [here](https://docs.seldon.io/projects/seldon-core/en/latest/analytics/analytics.html#custom-metrics).
```

Comment:
I have the feeling we should try to keep relative links wherever possible

Reply:
Somehow didn't see Clive's comment when browsing through changes. Sorry for duplicated!